### PR TITLE
Hw 2 boolean-expression solution

### PR DESCRIPTION
**Pull request by maks45**

### CODE REVIEW

File: src/main/java/core/basesyntax/BooleanExpression.java [hunk 1]
```diff
@@ -14,6 +14,9 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+
+        return a == b && a != c && a != d
+                || a == c && a != b && a != d
+                || a == d && a != c && a != b;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BooleanExpression.java]: try to solve this with XOR operator '^'. Google it)

Comment (maks45) [reply src/main/java/core/basesyntax/BooleanExpression.java]: done

File: src/main/java/core/basesyntax/BooleanExpression.java [hunk 2]
```diff
@@ -14,6 +14,7 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BooleanExpression.java]: remove empty line

Comment (maks45) [reply src/main/java/core/basesyntax/BooleanExpression.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**